protocols/raft/src/main/java/io/atomix/protocols/raft/storage/snapshot/impl/DefaultSnapshotStore.java

```diff
@@ -65,12 +63,12 @@ private void open() {
    */
   private Collection<Snapshot> loadSnapshots() {
     // Ensure log directories are created.
-    storage.directory().mkdirs();
+    directory.mkdirs();
 
     final List<Snapshot> snapshots = new ArrayList<>();
 
     // Iterate through all files in the log directory.
-    for (final File file : storage.directory().listFiles(File::isFile)) {
+    for (final File file : directory.listFiles(File::isFile)) {
```

Comment:
I think list files could return null - would be good to check this

Reply:
Sure, I didn't write this I was just refactoring it :sweat_smile: